core/shared/src/main/scala/zio/stm/TPriorityQueue.scala

```diff
@@ -30,6 +30,12 @@ import scala.collection.immutable.SortedMap
  */
 final class TPriorityQueue[A] private (private val ref: TRef[SortedMap[A, ::[A]]]) extends AnyVal {
 
+  /**
+   * Checks whether the queue is empty.
+   */
+  def isEmpty: USTM[Boolean] =
```

Comment:
do you normally implement `nonEmpty` in similar collections?

Reply:
We don't yet, but we could provide them separately.

Reply:
We haven't on a lot of these historically but I don't think there is any reason not to. I just added it to this PR for `TPriorityQueue`.